### PR TITLE
Qt 5.4.2

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: Library/Formula/qt5.rb [hunk 1]
```diff
@@ -11,9 +11,9 @@ def message; <<-EOS.undent
 
 class Qt5 < Formula
   homepage "https://www.qt.io/"
-  url "https://download.qt.io/official_releases/qt/5.4/5.4.1/single/qt-everywhere-opensource-src-5.4.1.tar.xz"
-  mirror "http://qtmirror.ics.com/pub/qtproject/official_releases/qt/5.4/5.4.1/single/qt-everywhere-opensource-src-5.4.1.tar.xz"
-  sha256 "1b7eb91e153176ac917f72b6bf443f987abf47c4208cdd43e2307684a7fad860"
+  url "https://download.qt.io/official_releases/qt/5.4/5.4.2/single/qt-everywhere-opensource-src-5.4.2.tar.xz"
+  mirror "http://qtmirror.ics.com/pub/qtproject/official_releases/qt/5.4/5.4.2/single/qt-everywhere-opensource-src-5.4.2.tar.xz"
```
Comment (vszakats) [inline Library/Formula/qt5.rb]: I'd suggest changing `mirror` to this one, which does support ssl/tls:
https://www.mirrorservice.org/sites/download.qt-project.org/official_releases/qt/5.4/5.4.2/single/qt-everywhere-opensource-src-5.4.2.tar.xz

Comment (tdsmith) [inline Library/Formula/qt5.rb]: Please don't push a new commit for this, because this is taking forever on the bots; I'll fix it up when I pull.

**Comment by tdsmith** [comment]
Timed out -- but only on Mountain Lion -- at 120 minutes, qq. We could pull this without a Mountain Lion bottle... or I'll kick off a longer build early tomorrow PT, which seems like a pretty quiet time for the bot.

**Comment by DomT4** [comment]
I think it took about 2hrs40m on ML the last time around.

**Comment by mikemcquaid** [comment]
Kicked off a rebuild, set timeout to a hilariously large number.

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for simple version updates is `qt5 5.4.2`.

**Pull request closed without merging**